pkg/common/kafka/admin/admin_kafka.go

```diff
@@ -56,11 +56,12 @@ func NewKafkaAdminClient(ctx context.Context, namespace string) (AdminClientInte
 		return nil, err
 	}
 
-	// Should Only Be One Kafka Secret In The Namespace
+	// Handle Various Numbers Of Kafka Secrets - Currently Only Support One!
 	if len(kafkaSecrets.Items) != 1 {
```

Comment:
Comment is a little weird, if it only supports 1.

Reply:
yep - the implementation evolved out from under the comment - will clarify ; )